.changeset/long-terms-deny.md

```diff
@@ -0,0 +1,17 @@
+---
+"fluid-framework": minor
+"@fluidframework/tree": minor
+"__section": tree
+---
+Move "Unsafe" `@system` types to `System_Unsafe` namespace
+
+Working code conforming to the [rules regarding API Support Levels](https://fluidframework.com/docs/build/releases-and-apitags#api-support-levels) should be unaffected, but this resolves an issue which required violating these rules and directly referencing `@system` types.
+
+In some cases, packages exporting SharedTree schema related types for recursive schema could hit errors like:
+
+> `error TS2742: The inferred type of 'YourSchema' cannot be named without a reference to '../node_modules/@fluidframework/tree/lib/internalTypes.js'. This is likely not portable. A type annotation is necessary.`
```

Comment:
Nit: should this be in a `\`\`\`shell` block instead?

Reply:
I don't think so. This is a string literal, so syntax highlighting it like a shell command wouldn't be right. At least I hope people don't pipe errors from TSC into their shells...

Reply:
Oh, fair. I was viewing it as console output, but it would also appear in intellisense.